conf.yaml

```diff
@@ -386,7 +376,7 @@ contents:
               - title:      Enterprise Search Node.js client
                 prefix:     enterprise-search-node
                 current:    *stackcurrent
-                branches:   [ 8.x, 8.18, 8.17, 8.16, 8.15, 8.14, 8.13, 8.12, 8.11, 8.10, 8.9, 8.8, 8.7, 8.6 ]
+                branches:   [ 8.19, 8.18, 8.17, 8.16, 8.15, 8.14, 8.13, 8.12, 8.11, 8.10, 8.9, 8.8, 8.7, 8.6 ]
```

Comment:
So while I did rename the branch here for consistency, I don't think we expect to release an 8.19 release for any enterprise search clients. Should we exclude 8.19 for all Enterprise Search clients, or is it useful to keep in 8.19 the mention that it was deprecated in 8.18 and won't see any other release?

Reply:
Good question is there no 8.19 release for enterprise search?